### ISSUE TITLE
2 contactpersonen in invoerscherm overige data maken

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/Onderzoeksbibliotheek#8]
Er is nu 1 naam van een contactpersoon in het invoerscherm, dat moeten er 2 worden. 
Van deze personen zowel NAAM als EMAIL op laten geven.

nb: extra velden mogen ook leeg gelaten worden.

1: naam technische contactpersoon
2: naam inhoudelijke contactpersoon

De huidige contactpersonen mogen allemaal worden: technisch contactpersoon
zie ook: https://github.com/IDgis/Onderzoeksbibliotheek/issues/7

**Comment by HermanAssink** [comment]
@gerardnienhuis: Kunnen we er vanuit gaan dat deze aanpassing, evenals #7 , betrekking heeft op alle varianten (WOO-bank, Overige data en Onderzoeksbibliotheek) zodat we met één versie van programmatuur kunnen blijven werken?

**Comment by gerardnienhuis** [comment]
ja, bedoeling is dat de software van onderzoeksbibiotheek en de WOO zoveel mogelijk gelijk blijft, dus zeker wat rapportfuncties betreft

**Comment by gerardnienhuis** [comment]
merk op: invoerscherm voor nieuwe documenten voor WOO, onderzoeksbank en "overige data" is dus hetzelfde. Blijft vergelijkbaar met huidige functionaliteit. Er komt alleen voor de WOO een extra optie bij. De "schakel" optie is nu al  "type" onderzoek, dat verandert niet. Dus, is er niets aangeklikt, dan wordt dit "overige data", en wordt er Wet Open Overheid aangeklikt, dan betreft het Wet Open Overheid en wordt er een ander type onderzoek aangeklikt, dan wordt dit informatie voor de onderzoeksbank.
